src/backend/executor/nodeDML.c

```diff
@@ -36,6 +36,28 @@ ExecDMLExplainEnd(PlanState *planstate, struct StringInfoData *buf)
 	planstate->instrument->execmemused += DML_MEM;
 }
 
+/*
+ * Edit input attr numbers of projection using attributes map
+ */
+void
+RemapProjection(ProjectionInfo *projInfo, AttrMap *map)
+{
+	int *varNumbers = projInfo->pi_varNumbers;
+	int numSimpleVars = projInfo->pi_numSimpleVars;
+
+	for (int i = 0; i < numSimpleVars;++i)
+	{
+		varNumbers[i] = attrMap(map, varNumbers[i]);
+	}
```

Comment:
The surrounding code doesn't use braces for 1-line code blocks, so we should also follow this way to keep code style consistent (though personally I'd prefer to keep braces :) ).
```suggestion
	for (int i = 0; i < numSimpleVars; ++i)
		varNumbers[i] = attrMap(map, varNumbers[i]);
```

Reply:
done